src/Symfony/Component/Validator/Constraints/NullValidator.php

```diff
@@ -11,25 +11,12 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
-use Symfony\Component\Validator\Constraint;
-use Symfony\Component\Validator\ConstraintValidator;
-
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated Deprecated since version 2.3.28, to be removed in 3.0. Use
```

Comment:
We do not add deprecations in patch version. So this comment should should only be added in 2.7.